<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>571</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Czytelnicy">
     <attribute name="title">
      <string>Czytelnicy</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>751</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Wyswietl_Czytelnicy">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Wyswietl</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Imie_Czytelnicy">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>310</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Nazwisko_Czytelnicy">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>350</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>320</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Imie:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>350</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nazwisko:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Dodaj_Czytelnicy">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>390</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Ksiazki">
     <attribute name="title">
      <string>Ksiazki</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Wypozycz">
     <attribute name="title">
      <string>Wypozycz</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
